tests/integration/shared/sources/new-or-changed-sources-test.js

```diff
@@ -87,9 +87,9 @@ test('Live crawl and scrape', async t => {
   for (const key of sourceKeys) {
     let crawlCompleted = false
     try {
-      await crawlSource({ source: key })
+      await { crawl }({ source: key })
```

Comment:
That's funky, I didn't even know that kind of thing was possible. 👍

Reply:
It's probably not, I might have fat-fingered it. :////